Adding Wollok package

javierfernandes opened the pull request:
Adds utilities for Wollok Programming Language (https://github.com/uqbar-project/wollok)

FichteFoll commented:
What is the purpose of the `SublimeLinter-contrib-wchecker` folder? As it stands, Sublime Text will not load any plugin files from that folder. I suggest adding this as a separate package to https://github.com/SublimeLinter/package_control_channel.

You can remove `packages.json`, it won't be looked at.

javierfernandes commented:
Ok done.
* Removed the packages.json
* Removed the SublimeLinter-contrib-wchecker linter (moved to a new repo for which I will create a new fork and PR to be included in SublimeLinter)
* Created new tag 0.0.2

Sorry for the delay !
